lua/modules/plugins/tool.lua

```diff
@@ -62,6 +62,7 @@ tool["gelguy/wilder.nvim"] = {
 tool["nvim-telescope/telescope.nvim"] = {
 	lazy = true,
 	cmd = "Telescope",
+	event = "UIEnter",
```

Comment:
```suggestion
	event = "LazyVimStarted",
```

I think Telescope can be loaded after UIEnter.

Reply:
oh nvm discard this comment